ipmininet/router/config/zebra.py

```diff
@@ -55,6 +56,7 @@ class Zebra(QuaggaDaemon):
     # We want zebra to preserve existing routes in the kernel RT (e.g. those
     # set via ip route)
     STARTUP_LINE_EXTRA = '-k'
+    last_routerid = ip_interface("0.0.0.1")
```

Comment:
As all you want is an IP address you should be good with ip_address (or IPv4Address) directly